frameworks/hdfs/integration/tests/test_utils.py

```diff
@@ -0,0 +1,183 @@
+import os
```

Comment:
Filed INFINITY-761 at top of backlog.  It's not totally trivial.  Didn't want to lose what focus there is for this PR.